src/System.CommandLine.Tests/UseHelpTests.cs

```diff
@@ -377,6 +377,25 @@ public void Layout_can_be_composed_dynamically_based_on_context()
             customOutput.Out.ToString().Should().Be($"Custom layout!{NewLine}{NewLine}{GetDefaultHelp(commandWithCustomHelp, false)}");
         }
 
+        [Fact]
+        public void Help_sections_can_be_wrapped()
+        {
+            var parser = new CommandLineBuilder()
+                         .UseHelp(ctx => ctx.HelpBuilder.CustomizeLayout(CustomLayout))
+                         .Build();
+
+            var console = new TestConsole(windowWidth: 10);
+            parser.Invoke("-h", console);
+
+            string result = console.Out.ToString();
+            result.Should().Be($"  123  123{NewLine}  456  456{NewLine}  78   789{NewLine}       0{NewLine}{NewLine}{NewLine}");
```

Comment:
We usually test this by creating a `HelpBuilder` with the desired width. I'm not sure I'd want to introduce an interface to enable testing at this higher layer when we can already test it lower down.

Reply:
well, the reason it was not caught is because it uses `HelpBuilder` for tests. The issue was actually when instantiating `HelpBuilder`.

Reply:
Can you think of an approach that makes this testable without introducing the interface?